internal/cleanup/cleanup.go

```diff
@@ -22,6 +22,8 @@ import (
 	"time"
 
 	"github.com/google/exposure-notifications-server/internal/database"
+	exportdb "github.com/google/exposure-notifications-server/internal/export/database"
```

Comment:
I see that here you alias exportdb whereas in other files you alias coredb. Is there any logic to this? Seems like it would be better to keep consistent with what is aliased or not.